android/src/main/java/com/swmansion/reanimated/MapUtils.java

```diff
@@ -0,0 +1,72 @@
+package com.swmansion.reanimated;
+
+import com.facebook.react.bridge.Dynamic;
+import com.facebook.react.bridge.JSApplicationCausedNativeException;
+import com.facebook.react.bridge.NoSuchKeyException;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.ReadableMap;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class MapUtils {
+  public static boolean getBoolean(ReadableMap map,  @Nonnull String name, String errorMsg) {
```

Comment:
I don't know, but maybe we can do sth more generic:


like 
```java
public static boolean obtain(ReadableMap map, @Nonnull ReadableType type, @Nonnull String name, String errorMsg) 
```

Reply:
Or maybe no. I want to omit `Integer` casting also if not needed 🤔.

Reply:
It's problematic due to Number -> Int | Double case scenario.